benchmarks/testminer.py

```diff
@@ -694,3 +700,73 @@ def get_result_data(self, test_job_id):
             return (None, None)
         return (db_attachments[0][0], base64.b64decode(db_attachments[0][1]))
 
+
+class AndroidMultinodeBenchmarkResults(LavaTestSystem):
+    def __init__(self, *args):
+        self.host_test_id = "lava-android-benchmark-host"
+        super(AndroidMultinodeBenchmarkResults, self).__init__(*args)
+
+    def get_test_job_results(self, test_job_id):
+        if self.host_test_id == None:
+            return []
+        logger.debug("Multinode measurement parsing from: {0}".format(self.host_test_id))
+
+        status = self.call_xmlrpc('scheduler.job_status', test_job_id)
+
+        if not ('bundle_sha1' in status and status['bundle_sha1']):
+            return []
+
+        test_result_list = []
+
+        sha1 = status['bundle_sha1']
+        result_bundle = self.call_xmlrpc('dashboard.get', sha1)
+        bundle = json.loads(result_bundle['content'])
+
+        target = [t for t in bundle['test_runs'] if t['test_id'] == 'multinode-target']
+        if target:
+            target = iter(target).next()
+        else:
+            return test_result_list
+        host = [t for t in bundle['test_runs'] if t['test_id'] == self.host_test_id]
+        if host:
+            host = iter(host).next()
+        else:
+            return test_result_list
+        src = (s for s in host['software_context']['sources'] if 'test_params' in s).next()
```

Comment:
it would be nice to be consistent in the usage of list comprehensions (i.e. [] vs ())